app/src/main/java/com/tagalong/ServerRequests.java

```diff
@@ -37,11 +39,21 @@ public void storeUserDataInBackground(User user, GetUserCallback callback) {
 		progressDialog.show();
 		new StoreUserDataAsyncTask(user, callback).execute();
 	}
+
+	public void storeEventDataInBackground(Event event, GetEventCallback callback) {
```

Comment:
@cpf5193 it's weird because it looks fine in my android studio. Either I fixed it sometime or github is displaying it oddly